apps/spruce/src/pages/waterfall/index.tsx

```diff
@@ -21,6 +20,7 @@ import WaterfallSkeleton from "./WaterfallSkeleton";
 
 const Waterfall: React.FC = () => {
```

Comment:
might be a dumb question but do we want to do a minor release on one of these PRs? not sure since there are multiple related PRs that are going in before tomorrow